pynos/versions/base/interface.py

```diff
@@ -3315,16 +3315,17 @@ def vrrpe_vmac(self, **kwargs):
             config.find('.//*virtual-mac').set('operation', 'delete')
         return callback(config)
 
-    @property
-    def ve_interfaces(self):
+    def ve_interfaces(self, **kwargs):
         """list[dict]: A list of dictionary items describing the operational
         state of ve interfaces along with the ip address associations.
         """
         urn = "{urn:brocade.com:mgmt:brocade-interface-ext}"
-        int_ns = 'urn:brocade.com:mgmt:brocade-interface-ext'
 
+        rbridge_id = None
+        if 'rbridge_id' in kwargs:
+            rbridge_id = kwargs.pop('rbridge_id')
```

Comment:
you can change this to rbridge_id = kwargs.pop('rbridge_id', None) and remove the 2 lines above